packages/core/src/amazonqFeatureDev/errors.ts

```diff
@@ -57,6 +57,23 @@ export class SelectedFolderNotInWorkspaceFolderError extends ToolkitError {
     }
 }
 
+export class PromptRefusalException extends ToolkitError {
+    constructor() {
+        super(
+            'I\'m sorry, I can\'t generate code for your request. Please make sure your message and code files comply with the <a href="https://aws.amazon.com/machine-learning/responsible-ai/policy/">AWS Responsible AI Policy.</a>',
```

Comment:
Just to confirm if these error msgs got reviewed by doc writers.

Reply:
Yup, I got this extract from the doc writer errors string review doc. I can DM you with the link,